statsmodels/tsa/holtwinters.py

```diff
@@ -0,0 +1,444 @@
+"""
+Created on Wed Jul 12 09:35:35 2017
+
+@author: tvzyl
+"""
+import numpy as np
+import pandas as pd
+import warnings
+
+from statsmodels.compat.python import iteritems
+
+from scipy.stats import boxcox
+try:
+    from scipy.special import inv_boxcox
+except ImportError:
+    inv_boxcox = lambda x, lmbda: np.exp(np.log1p(lmbda * x) / lmbda) if lmbda != 0 else np.exp(x)
```

Comment:
Should be a def function.    No assigning lambdas.

Reply:
done